input/pagecontent/etlTalent.md

```diff
@@ -0,0 +1,216 @@
+    patient.setMeta(m);
+
+    switch (geschlecht){
+    case "male": patient.setGender(AdministrativeGender.MALE); break;
+    case "female": patient.setGender(AdministrativeGender.FEMALE); break;
+    default: patient.setGender(AdministrativeGender.UNKNOWN);
+    }
+
+    patient.setId(cPatId);
+
+    if(geburtsDatum!=null)
+      patient.setBirthDate(geburtsDatum);
+
+    if(sterbeDatum!=null)
+      patient.setDeceased(new DateTimeType(sterbeDatum));
+
+    patientMap.put(patient.getId(),patient);
+    System.err.println("size after addition: "+patientMap.size());
+
+    bundle.addEntry()
+       .setFullUrl(patRand.getValueAsString())
+       .setResource(patient)
+       .getRequest()
+       .setUrl("Patient/"+patient.getId())
+       .setMethod(HTTPVerb.PUT);
+
+    patientToBundle.put(patient.getId(), bundle.getId());
+
+### 5. Create and assign the diagnosis ressources:
+    Condition condition = new Condition();
+    condition
+       .getCode()
+       .addCoding()
+       .setSystem("http://fhir.de/CodeSystem/dimdi/icd-10-gm")
+       .setVersion("2016")
+       .setCode(code)
+       .setDisplay(display.trim());
+    condition.setId(recId);
+    if(obEncId!=null)
+      condition.setEncounter(new Reference("Encounter/fn"+obEncId));
+    if(onset!=null)
+      condition.setOnset(new DateTimeType(onset));
+    if(abatement!=null)
+      condition.setAbatement(new DateTimeType(abatement));
+    condition.getClinicalStatus()
+       .addCoding()
+       .setSystem("http://terminology.hl7.org/CodeSystem/condition-clinical")
+       .setCode("resolved");
+
+    condition.setSubject(new Reference("Patient/"+patientMap.get(obPatId).getId()));
+
+	Bundle lookedUpBundle=bdlMap.get(patientToBundle.get(obPatId));
+
+    lookedUpBundle.addEntry()
+       .setResource(condition)
+       .setFullUrl(IdDt.newRandomUuid().getValueAsString())
+       .getRequest()
+       .setUrl("Condition/"+condition.getId())
+       .setMethod(HTTPVerb.PUT);
+
+    System.err.println("condition added.");
+
+### 6. Upload bundles:
+    FhirContext ctx = FhirContext.forR4();
+
+    // Create a client and PUT the transaction to the server
+    IGenericClient client = ctx.newRestfulGenericClient("http://FHIRSERVER:8080/fhir");
+
+    // Create an HTTP basic auth interceptor
+    String username = "admin";
+    String password = "psw";
+    IClientInterceptor authInterceptor = new BasicAuthInterceptor(username, password);
+    client.registerInterceptor(authInterceptor);
+
+    System.err.println("number of bundles: "+bdlMap.size());
+    for (Bundle bdlIt : bdlMap.values()) {
+      System.out.println(ctx.newJsonParser().setPrettyPrint(true).encodeResourceToString(bdlIt));
+      Bundle resp = client.transaction().withBundle(bdlIt).execute();
+      System.out.println(ctx.newJsonParser().setPrettyPrint(true).encodeResourceToString(resp));
+    }
+
+![TOS Flow1](../images/TOS_FHIR_02_Flow1.png)
+
+![TOS Flow2](../images/TOS_FHIR_03_Flow2.png)
+
+### Appendix:
```

Comment:
Make this h2 heading so it is not grouped under "In details"

Reply:
DONE